CMakeLists.txt

```diff
@@ -123,8 +124,11 @@ set(SWIFT_INSTALL_COMPONENTS "${_SWIFT_KNOWN_INSTALL_COMPONENTS}" CACHE STRING
 # * toolchain-dev-tools -- install development tools useful in a shared toolchain
 # * dev -- headers and libraries required to use Swift compiler as a library.
 
-set(SWIFT_SDKS "" CACHE STRING
-    "If non-empty, limits building target binaries only to specified SDKs (despite other SDKs being available)")
+set(SWIFT_CROSS_COMPILE_STDLIB_TARGETS "" CACHE STRING
+    "A list of deployment-targets to cross-compile the standard-library for (by default, the standard library is always compiled for the host)")
+
+set(SWIFT_SKIP_HOST_STDLIB "" CACHE STRING
+  "If True, will not build the standard library for the host")
```

Comment:
This doesn't actually work right now, but a couple of parts of the old CMakeLists (which was hardcoded so this would never happen) indicated that it was supported at some point, or would at least be desirable in the future.

Specifically, `SWIFT_PRIMARY_VARIANT` gets set to Android when not building the stdlib for the Linux host, and swift-reflection-dump is gated by checking if we're building for the host's stdlib.